Sources/account/ALRegisterUserClientService.m

```diff
@@ -454,6 +454,20 @@ - (void)logoutWithCompletionHandler:(void(^)(ALAPIResponse *response, NSError *e
     }];
 }
 
+// Use this only when Logout APi Fails due to authorization
+- (void)logoutUserLocally {
```

Comment:
Call this function from `logoutWithCompletionHandler` or create another private function and call from both the functions to avoid duplicating the logic.

Reply:
created a new private function & called it from both places